rest/view_api_test.go

```diff
@@ -50,44 +57,47 @@ func TestDesignDocs(t *testing.T) {
 }
 
 func TestViewQuery(t *testing.T) {
+
 	var rt RestTester
+	defer rt.Close()
+
 	response := rt.SendAdminRequest("PUT", "/db/_design/foo", `{"views":{"bar": {"map": "function(doc) {emit(doc.key, doc.value);}"}}}`)
 	assertStatus(t, response, 201)
 	response = rt.SendRequest("PUT", "/db/doc1", `{"key":10, "value":"ten"}`)
 	assertStatus(t, response, 201)
 	response = rt.SendRequest("PUT", "/db/doc2", `{"key":7, "value":"seven"}`)
 	assertStatus(t, response, 201)
 
-	response = rt.SendAdminRequest("GET", "/db/_design/foo/_view/bar", ``)
-	assertStatus(t, response, 200)
-	var result sgbucket.ViewResult
+	result, err := rt.WaitForNAdminViewResults(2, "/db/_design/foo/_view/bar")
```

Comment:
Explain why the wait is needed here, and that it could be changed to stale=false.  (and then there would only be a wait needed after the design doc creation)